mDNSWindows/SystemService/Service.c

```diff
@@ -29,7 +29,7 @@
 #include	"uds_daemon.h"
 #include	"GenLinkedList.h"
 #include	"Service.h"
-#include	"EventLog.h"
+#include	"mDNSWindows/SystemService/EventLog.h"
```

Comment:
This should be #ifdef __MINGW__ ... #else ... #endif, so that it builds on Visual Studio as well. Better still, fix the MinGW Makefile to add a "-I mDNSWindows/SystemService" so we don't need the ifdef.

Reply:
This might be Android buildsystem related issue. Let's leave it out for now.